frappe/core/doctype/user/user.py

```diff
@@ -837,3 +863,10 @@ def extract_mentions(txt):
 	"""Find all instances of @username in the string.
 	The mentions will be separated by non-word characters or may appear at the start of the string"""
 	return re.findall(r'(?:[^\w]|^)@([\w]*)', txt)
+
+
+def handle_password_test_fail(result):
+	suggestions = result['feedback']['suggestions'][0] if result['feedback']['suggestions'] else ''
+	warning = result['feedback']['warning'] if 'warning' in result['feedback'] else ''
+	suggestions += "<br/>Hint : Include Underscores, Numbers and Capital Letters in the password<br/>Eg: Eastern_43A1W"
```

Comment:
this string must also be translatable! 

hint: break it up into html and text parts and wrap `_("")`